### ISSUE TITLE
preset: 

### ISSUE THREAD

**Issue by BigCityCat** [GradienceTeam/Community#52]
### Issue

Gradience does not apply any saved preset. Log out or reboot. No preset can be applied. All settings are saved correctly. Flatseal settings used all applications. Using Adwaita.

### OS

Ubuntu Kinetic Kudu

### Version

0.3.0

### Installation method

Flatpak from Flathub

### Did you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/GradienceTeam/Gradience/blob/main/CODE_OF_CONDUCT.md).

**Comment by 0xMRTT** [comment]
Could you check if there is ~/.config/{gtk-3.0,gtk-4.0}/gtk.css ?

**Comment by BigCityCat** [comment]
Yes these files do exist. I will say that I have been using the Material You Gnome Shell Extension with success. I have it switched off when using Gradience. When using the Gradience application the theme appears to be applied but will not apply to any other applications.

**Comment by tfuxu** [comment]
@BigCityCat 
Please move this issue to the Gradience repository: https://github.com/GradienceTeam/Gradience/issues
